packages/grid/src/features/standard-table/components/StandardTableHeadItem.tsx

```diff
@@ -60,6 +60,10 @@ export const StandardTableHeadItem = React.memo(
       <th
         style={{
           background: background ?? "white",
+          borderBottom:
+            !disableSorting && selected
+              ? "1px solid var(--lhds-color-blue-500)"
+              : undefined,
```

Comment:
The problem with having a border in one case and undefined in the other is that the content shifts up and down. I would have the border always be there but transparent if not selected. I would also add a transparent top border to even things out.

Reply:
Also, we have 2px in our home-made grid. I checked with the design team if we should sync this somehow and they think we should use 2px here as well so please up it :)

Reply:
Also, verify that it looks fine when using sticky header, sticky columns, etc.